basiclti/tsugi-util/src/test/org/tsugi/basiclti/BasicLTIUtilTest.java

```diff
@@ -97,12 +97,41 @@ public void testOAuthGET()
 	public void iso8601Text() throws Exception {
 		String x = BasicLTIUtil.getISO8601();
 		assertTrue(x.contains("Z"));
+		assertTrue(x.contains("T"));
 		String target = "2017-08-20:10:00:00";
+		String iso8601 = "2017-01-20T10:00:00Z";
```

Comment:
should try out the standard form:

2022-08-13T11:01:41+00:00

2022-08-13T11:01:41-04:00

Reply:
I do test these forms in the loop variation of the test.

            "2017-01-20T11:00:00+01:00",
            "2017-01-20T09:00:00-01:00",